src/config/serverConfig.ts

```diff
@@ -1,12 +1,16 @@
 import { constants, ServerConfig } from '@projecttacoma/node-fhir-server-core';
 import { MeasureService } from '../services';
+import { LibraryService } from '../services/LibraryService';
```

Comment:
Suggestion: just add `export * from './LibraryService'` to `services/index.ts`, and that will allow you to combine these imports into the one on line 2